README.md

```diff
@@ -0,0 +1,98 @@
+# ss18_showmaster
+Roboy wishes to be more entertaining and impressive. With his new skills he is reliably able to attract people to its presence independent to their relation to him. He appeals as well to an audience as to a group of developers, satisfying both of their preferences.
```

Comment:
this is a very confusing sentence. please rewrite in simple words. the main message: "roboy is fun and easy to control during fairs and other events with audience"